### PR TITLE
Add waiting view to hostname screen/layout

### PR DESCRIPTION
**Pull request by philipbrito**
Like the Login screen does, this PR adds the waiting view to hostname screen, thus, the user will be able to see that the connection is in progress.

### CODE REVIEW

File: app/src/main/res/layout/fragment_input_hostname.xml [hunk 1]
```diff
@@ -1,63 +1,79 @@
...
                 android:layout_width="0px"
                 android:layout_height="wrap_content"
                 android:layout_weight="1"
                 android:orientation="vertical">
 
-            <TextView
+                <TextView
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
+                    android:paddingStart="4dp"
```
Comment (matheusjardimb) [inline app/src/main/res/layout/fragment_input_hostname.xml:33]: should this become move to @dimen/padding_4, similar to @dimen/margin_8 used in this very same file?

Comment (matheusjardimb) [reply app/src/main/res/layout/fragment_input_hostname.xml:33]: not sure if that make sense :)

**Comment by matheusjardimb** [comment]
Isn't it weird that most layout (xml) files use mixed tab and spaces as indentation? 
It becomes really annoying when reviewing code changes ... :disappointed:

File: app/src/main/res/layout/fragment_input_hostname.xml (hunk 1 above)
Comment (philipbrito) [reply app/src/main/res/layout/fragment_input_hostname.xml:33]: It is a workaround to align the TextView w/ the EditText. Take a look:

Before
before

After:
after

**Comment by philipbrito** [comment]
Should we use [intellij-java-google-style.xml](https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml) @matheusjardimb ? This [file](https://github.com/RocketChat/Rocket.Chat.Android/blob/develop/config/quality/style-guide/intellij-google-style.xml) seems to refer to it, but is invalid.

**Comment by matheusjardimb** [comment]
Sure, I guess that should help removing the noise indentation can generate. 
In this case, I'd be great to have some instructions in the README file in order to help new contributors use this style. Know what I'm sayin?

**Comment by philipbrito** [comment]
I agree! I'll do that!

File: app/src/main/res/layout/fragment_input_hostname.xml (hunk 1 above)
Comment (matheusjardimb) [reply app/src/main/res/layout/fragment_input_hostname.xml:33]: I see. Not big deal tho

**Review by matheusjardimb — APPROVED** [review]

**Pull request merged**